container-chains/templates/frontier/runtime/src/lib.rs

```diff
@@ -165,6 +165,24 @@ pub type Executive = frame_executive::Executive<
     AllPalletsWithSystem,
 >;
 
+pub mod currency {
+    use super::Balance;
+
+    // Provide a common factor between runtimes based on a supply of 10_000_000 tokens.
+    pub const SUPPLY_FACTOR: Balance = 100;
```

Comment:
Did you copy this from moonbeam? I dont think this is needed here for now

Reply:
I mean the supply factor part

Reply:
I copied that from Tanssi actually
Should I delete it?